trinity/plugins/builtin/ethstats/ethstats_client.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+import datetime
+import json
+import typing
+
+import websockets
+
+from p2p.service import (
+    BaseService,
+)
+
+
+# Returns UTC timestamp in ms, used for latency calculation
+def timestamp_ms():
+    return round(datetime.datetime.utcnow().timestamp() * 1000)
+
+
+class EthstatsMessage(typing.NamedTuple):
+    command: str
+    data: dict
```

Comment:
This probably needs to be `Dict[str, Any]`

Reply:
Fixed this.